lte/gateway/c/session_manager/SessionCredit.cpp

```diff
@@ -319,6 +358,7 @@ GrantTrackingType SessionCredit::determine_grant_tracking_type(
   }
   bool tx_valid = grant.tx().is_valid();
   bool rx_valid = grant.rx().is_valid();
+  // TODO: add ALL_TOTAL_TX_RX case
```

Comment:
Tag a github username here

Reply:
A small nit, but this TODO should probably come before the 
```
if (grant.total().is_valid()) {
    return TOTAL_ONLY;
  }
```

Reply:
I decided TO DO it ;) 
so this will be done in this same commit soon